go.mod

```diff
@@ -1,3 +1,5 @@
 module git.helsinki.tools/helsinki-systems/wp4nix
 
 go 1.16
+
+require github.com/nix-community/go-nix v0.0.0-20231219074122-93cb24a86856 // indirect
```

Comment:
This shouldn't be an `indirect` dependency. Can you please run `go get -u ; go mod tidy` and push the changes? 😊